com.regnosys.rosetta.tests/src/com/regnosys/rosetta/generator/java/function/FunctionGeneratorTest.xtend

```diff
@@ -35,6 +35,55 @@ class FunctionGeneratorTest {
 	@Inject extension ModelHelper
 	@Inject extension ValidationTestHelper
 	
+	@Test
+	def void directlyUseAttributesOfImplicitVariableTest() {
+		val code = '''
+			namespace com.rosetta.test.model
+			version "${project.version}"
+			
+			type Foo:
+				a int (1..1)
+				b string (0..*)
+			
+			func F1:
+				inputs:
+					foos Foo (0..*)
+				output:
+					result int (0..*)
+				
+				add result:
+					foos
+						extract [ a + b count]
```

Comment:
We now never need to write `item -> ...`. You can just leave the `item` (or `it`, or whatever you want to call it) Hooray!